connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -245,34 +245,19 @@ public ConfigInfos validateConnectorConfig(Map<String, String> connectorConfig)
             ConfigDef basicConfigDef = (connector instanceof SourceConnector)
                                        ? SourceConnectorConfig.configDef()
                                        : SinkConnectorConfig.configDef();
-            Map<String, ConfigValue> validatedConnectorConfig = validateBasicConnectorConfig(
```

Comment:
I don't think you want to get rid of the `validateBasicConnectorConfig` call. The default just calls validate, but in `DistributedHerder` it also validates there won't be a conflict between the worker and consumer group for sink connectors.